### DISCUSSION TITLE
Frame processor face_enhancer not found

### DISCUSSION THREAD

**Discussion by Filthyfrankx** [hacksider/Deep-Live-Cam#519] [General]
any one see this when opening the run.py ?

**Comment by Filthyfrankx** [comment]
![image](https://github.com/user-attachments/assets/2e405136-5c9a-48e3-bb4f-b1184666edc9)

**Reply by fACE123123321** [reply]
Me too

**Comment by Filthyfrankx** [comment]
您发送的信件已经收到，我们会尽快给您答复。

**Comment by fACE123123321** [comment]
https://github.com/hacksider/Deep-Live-Cam/issues/162#issuecomment-2282640935 can help you

**Reply by dodgyrabbit** [reply]
Or this:
https://github.com/hacksider/Deep-Live-Cam/issues/191#issuecomment-2330258600

I think the most common problem is people install newer versions of torch and torchvision.

The reason it is failing, is because [gfpgan](https://github.com/TencentARC/GFPGAN/tree/master) has a dependency on [basicsr](https://github.com/XPixelGroup/BasicSR). Both of these are relatively old, and have a dependencies on torch and torchvision. The newer versions of torchvision changed their code a bit and breaks basicsr (and hence gfpgan fails to load properly, leading to the error).

Downgrade those packages to the versions as per [requirements.txt](https://github.com/hacksider/Deep-Live-Cam/blob/main/requirements.txt)
```
torch==2.0.1+cu118; sys_platform != 'darwin'
torch==2.0.1; sys_platform == 'darwin'
torchvision==0.15.2+cu118; sys_platform != 'darwin'
torchvision==0.15.2; sys_platform == 'darwin'
```
